datahub-web-react/src/app/entity/shared/utils.ts

```diff
@@ -97,6 +97,12 @@ function getGraphqlErrorCode(e) {
 
 export const handleBatchError = (urns, e, defaultMessage) => {
     if (urns.length > 1 && getGraphqlErrorCode(e) === 403) {
+        if (e.message === 'Only users granted permission to this tag can assign or remove it') {
+            return {
+                content: `${e.message}. The bulk edit being performed will not be saved.`,
+                duration: 3,
+            };
+        }
```

Comment:
why do we need this new message when we have a message about being unauthorized below?

Reply:
The new error message is for clarity to pinpoint which authorization is missing.